source/_components/zha.markdown

```diff
@@ -116,3 +137,28 @@ To remove modemmanager from an Debian/Ubuntu host run this command:
 ```bash
 sudo apt-get purge modemmanager
 ```
+
+### Can't connect to usb device and using docker
+If you can't connect as you are using docker you must forward your device from the host machine to the docker instance. This can be achieved by adding the device mapping to the end of the startup string or ideally using docker compose:
+#### Docker Compose
+Install docker-compose for your platform (rpi - `sudo apt-get install docker-compose`).
+
+Create a docker-compose.yml with the following data:
```

Comment:
```suggestion
Create a `docker-compose.yml` with the following data:

```

This is a bit scary. A new user might think he needs to create a new (additional) Docker Compose setup?

Reply:
I agree it is a bit scary, its more like an optional step. I've nested it in the trouble shooting section as it might not apply to all users, only those that choose to use docker.